bin/cli.js

```diff
@@ -3,17 +3,17 @@
 'use strict';
 
 const minimist = require('minimist');
-const _ = require('lodash');
+const camelCase = require('lodash.camelcase');
 const checkDependencies = require('../lib/check-dependencies');
 
 const argv = minimist(process.argv.slice(2));
 
 // camelCase the options
-for (const key in argv) {
+Object.keys(argv).forEach(key => {
```

Comment:
Performance shouldn't be a problem for `argv`, this list won't be very large.

What are the differences in behavior? Is it about the iteration order or something else?